contracts/VestedPayment.sol

```diff
@@ -0,0 +1,85 @@
+pragma solidity ^0.4.13;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./LifToken.sol";
+
+contract VestedPayment is Ownable {
+  using SafeMath for uint256;
+
+  // when the vested schedule starts
+  uint256 public startTime;
```

Comment:
what about `startTimestamp`?

Reply:
Whats the difference? I mean..its a time I think this name is good as it is.

Reply:
`startTime` is usually about the time in the day, like "it starts at 10:00am", timestamp is more clear in that it's date and time. But it's ok, feel free to leave it as `startTime` if you prefer :)